script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -1048,7 +1048,7 @@ Those are the criteria that he considers before reincarnating, but―――
 [sha:349676c88aa5a475fef26298aea30d7bca949f2e]{
 -- Page 733, Offset 26265.
 -- 理解してくれたか、<遠|・><野|・>志貴くん？」
-Do you see where I'm going with this, %{i}Tohno Shiki-kun?%{/i}"
+Do you see where I'm going with this, %{i}Tohno%{/i} Shiki-kun?"
```

Comment:
He put the emphasis on "Tohno" alone, implying that it's the high class family Roa's seeking.